clover/netcdf/variable.py

```diff
@@ -464,6 +464,35 @@ def datetimes(self):
         else:
             return numpy.array([datetime(*d.timetuple()[:6], tzinfo=pytz.UTC) for d in self.dates])
 
+    @property

```

Comment:
Looks like these characters are present on master as well. Trying to figure out how to get rid of them...